addons/mail/static/src/discuss/typing/common/typing.js

```diff
@@ -17,11 +17,15 @@ export class Typing extends Component {
     static props = ["channel?", "size?", "displayText?", "member?"];
     static template = "discuss.Typing";
 
+    get channel() {
+        return this.props.channel || this.props.member?.channelAsTyping?.channel;
+    }
```

Comment:
We actually don't need this fallback at all.

Either we receive a member and we display the typing of the member, and the channel is never used.

Or we receive a channel directly, and we just the received channel directly.